### PR TITLE
ClusterLoader - Fixing wait for controlling objects timeout

### PR DESCRIPTION
**Pull request by krzysied**
Fixing bug introduced in #386 . It caused setting timeout status even if there was no timeout.

### CODE REVIEW

File: clusterloader2/pkg/measurement/common/simple/wait_for_controlled_pods.go [hunk 1]
```diff
@@ -430,8 +430,12 @@ func (w *waitForControlledPodsRunningMeasurement) waitForRuntimeObject(clientSet
 				glog.Errorf("%s: error for %v: %v", w, key, err)
 				o.err = fmt.Errorf("%s: %v", key, err)
 			}
+			if o.status == timeout {
+				glog.Errorf("%s: %s timed out", w, key)
```
Comment (wojtek-t) [inline clusterloader2/pkg/measurement/common/simple/wait_for_controlled_pods.go]: glog error won't fail the test and we don't return error - I don't think it's expected.

Comment (krzysied) [reply clusterloader2/pkg/measurement/common/simple/wait_for_controlled_pods.go]: That's true. However the test failing is done by https://github.com/kubernetes/perf-tests/blob/61c121b65366dcb94efcfae54813190b3a78f1dc/clusterloader2/pkg/measurement/common/simple/wait_for_controlled_pods.go#L253

This file starts to look to complicated. I will think about how to rewrite it (in different PR).

Comment (wojtek-t) [reply clusterloader2/pkg/measurement/common/simple/wait_for_controlled_pods.go]: I don't think it's time to rewrite it now. I actually believe it shouldn't be a measurement.
So let's don't rewrite it now.

Comment (wojtek-t) [reply clusterloader2/pkg/measurement/common/simple/wait_for_controlled_pods.go]: Please add a comment why we don't need to return error here.

Comment (krzysied) [reply clusterloader2/pkg/measurement/common/simple/wait_for_controlled_pods.go]: Done.

**Pull request merged**